system/src/system_cloud_internal.cpp

```diff
@@ -522,11 +522,13 @@ int Spark_Save(const void* buffer, size_t length, uint8_t type, void* reserved)
 
 int Spark_Restore(void* buffer, size_t max_length, uint8_t type, void* reserved)
 {
-	size_t length = 0;
-	int error = HAL_System_Backup_Restore(0, buffer, max_length, &length, nullptr);
-	if (error)
-		length = 0;
-	return length;
+    // Force new session handshake for now
+    return 0;
```

Comment:
Do we want to merge this as is? or leave the session restoration problem for later